src/setup.py

```diff
@@ -27,6 +27,7 @@
     license='GPLv2+',
     packages=find_packages(),
     include_package_data=False,
+    package_data={ '': [ '../katello/contrib/etc/*/*/*.conf', '../katello/contrib/extra/*' ] },
```

Comment:
Have you considered moving `setup.py` to the top level? Not sure if that works or if it messes up the directory structure.

Reply:
I'm far from an expert of these setup.py stuff :-)  I tried to find a solution to package these files in a directory which is 'acceptable' and saw your solution with the README.md which I then used.
If you move the setup.py to top level, the content of the target installation would then contain src/katello instead of just katello. Maybe there are some tricks to fix this, too.

Reply:
> If you move the setup.py to top level, the content of the target installation would then contain src/katello instead of just katello. Maybe there are some tricks to fix this, too.
Perhaps this helps: https://github.com/pypa/sampleproject/blob/918bd331501de42be32666bd5140d04d00b39386/setup.py#L118=

It might be easier to use the manifest: https://packaging.python.org/en/latest/guides/using-manifest-in/ and overall https://packaging.python.org/en/latest/ can be quite useful.

Reply:
Thanks. I tired it, like this:
`
package_dir={'', 'src'`}
packages=find_packages(where='src')
`
This was possible but the etc/extra files are then not packaged or not in katello/contrib/ (which I would prefer).